jetbrains-core/tst/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtilsTest.kt

```diff
@@ -375,7 +374,6 @@ class EcsExecUtilsTest {
                     Task.builder().containers(
                         Container.builder()
                             .name(containerName)
-                            .runtimeId(containerRuntimeId)
```

Comment:
I think you can get rid of this whole stub - since we removed the call to `describeTasks`

Reply:
done